src/pretix/plugins/sendmail/views.py

```diff
@@ -40,7 +42,16 @@ def form_valid(self, form):
         for o in orders:
             try:
                 mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                     None, self.request.event, locale=o.locale, order=o)
+                     {
+                         'event': o.event,
+                         'order': o.code,
+                         'order_date': date_format(o.datetime, 'SHORT_DATETIME_FORMAT'),
```

Comment:
We *might* need to care about timezones here. Could you test that by setting the timezone in the event settings to something non-UTC and then test sending an email? If it is a problem, the solution should be along those lines:

```
tz = pytz.timezone(request.event.settings.timezone)
...
'order_date': date_format(o.datetime.astimezone(tz), 'SHORT_DATETIME_FORMAT')
```